packages/pcd-types/src/pcd.ts

```diff
@@ -1,74 +1,165 @@
+/**
+ * All PCDs consist of a "claim", which is the human-interpretable statement
+ * that the PCD is making (i.e. "I am a Zuzalu resident"); and a "proof" attached
+ * to the "claim," which is a cryptographic or mathematical proof of the claim.
```

Comment:
The sense in which I mean this data is meaningful is that it can be understood by a developer who looks at the claim, so that they can use it in their application. For example, a field like `likes` that has an integer value coming from some sort of Facebook PCD would be 'human readable'. Contrast this with something like a signature, which would be located in a `proof` field - that is not comprehensible to a human being.

Reply:
A count of likes is arguably human-readable, but something like a user ID which is 20 digit number isn't really human-readable.  It is a valid part of a claim, though.  I guess I'm interpreting "human" more as end-user not the app developer's code.